internal/stacks/stackplan/planned_change.go

```diff
@@ -277,18 +277,12 @@ func (pc *PlannedChangeResourceInstancePlanned) PlannedChangeProto() (*terraform
 		// if this plan later gets applied.
 		// We only emit a "raw" in this case, because this is a relatively
 		// uninteresting edge-case.
```

Comment:
I'm not sure this comment fits here any more, although I am having some trouble with GitHub's diff of this file…

Reply:
Moved the comments around so they line up properly again.